src/spartan/polys/multilinear.rs

```diff
@@ -63,8 +63,12 @@ impl<Scalar: PrimeField> MultilinearPolynomial<Scalar> {
     self.Z.len()
   }
 
+  /// Returns true if no evaluations.
```

Comment:
Note for posterity: removed from upstream as unused code in https://github.com/microsoft/Nova/pull/257, I agree with its re-introduction here because of [len_without_is_empty](https://rust-lang.github.io/rust-clippy/master/index.html#/len_without_is_empty)